data/preset/buffet.yaml

```diff
@@ -307,22 +307,6 @@ defines:
             show-titles: false
             show-toc: false
             custom-css: buffet-custom.css
-        nav-content:
```

Comment:
I don't think we necessarily need to remove this one. It's a lot smaller than the 'see more' popup and doesn't share many of the same problems.